src/core/DataHelper.ttslua

```diff
@@ -723,6 +723,89 @@ LOCATIONS_DATA = JSON.decode(
   "Poker Table": {"type": "perPlayer", "value": 1, "clueSide": "back"},
   "Baccarat Table": {"type": "perPlayer", "value": 1, "clueSide": "back"},
 
+  "Rainy London Streets": {"type": "perPlayer", "value": 2, "clueSide": "back"},
+  "Big Ben": {"type": "perPlayer", "value": 1, "clueSide": "back"},
+  "Traitors' Gate": {"type": "perPlayer", "value": 1, "clueSide": "back"},
+  "The Tower Bridge": {"type": "perPlayer", "value": 1, "clueSide": "back"},
+  "Kensington Gardens": {"type": "perPlayer", "value": 1, "clueSide": "back"},
+  "Tower of London": {"type": "perPlayer", "value": 1, "clueSide": "back"},
+  "Westminster Abbey": {"type": "perPlayer", "value": 1, "clueSide": "back"},
+  "Tower Prison": {"type": "perPlayer", "value": 2, "clueSide": "back"},
+
+  "Saadian's Tombs_b9fc93": {"type": "perPlayer", "value": 3, "clueSide": "back"},
```

Comment:
Why do some have guids and others don't?

Reply:
GUIDs are only needed for locations that exist multiple times